{"name": "test_error_cases[None-5-add-\\u041f\\u0443\\u0441\\u0442\\u044b\\u0435 \\u0437\\u043d\\u0430\\u0447\\u0435\\u043d\\u0438\\u044f]", "status": "passed", "steps": [{"name": "Открываем и заполняем форму с ошибками", "status": "passed", "start": 1738868332653, "stop": 1738868333019}, {"name": "Проверяем сообщение об ошибке", "status": "passed", "start": 1738868334019, "stop": 1738868334061}], "parameters": [{"name": "num1", "value": "None"}, {"name": "num2", "value": "5"}, {"name": "operation", "value": "'add'"}, {"name": "expected_error", "value": "'Пустые значения'"}], "start": 1738868332653, "stop": 1738868334061, "uuid": "14f0b97f-721c-4f20-80df-e47fbf4f1a54", "historyId": "17eaa3b5de455a41922f0d40adf35c22", "testCaseId": "938b571ad6ce813f24f8a764b084d3a2", "fullName": "tests.test_calculator.TestAdvancedCalculator#test_error_cases", "labels": [{"name": "story", "value": "Проверка обработки ошибок"}, {"name": "feature", "value": "Тесты калькулятора"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_calculator"}, {"name": "subSuite", "value": "TestAdvancedCalculator"}, {"name": "host", "value": "anton"}, {"name": "thread", "value": "28068-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_calculator"}]}